content/en/monitors/incident_management.md

```diff
@@ -18,19 +16,55 @@ In the Datadog paradigm, any of the following are appropriate situations for dec
 * An issue is or may be impacting customers or services.
 * You do not know whether you should call an incident. Notify other people and increase severity appropriately.
 
-<div class="alert alert-warning">
-<a href="https://app.datadoghq.com/incidents">Incident Management</a> is currently in beta and available to all customers. For more information, reach out to sales@datadoghq.com.
-</div>
-
 ## Usage
 
-Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page for a feed of all ongoing incidents. You can configure additional fields that appear for all incidents in [Incident Settings][2].
+Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page to see a feed of all ongoing incidents.  You can configure additional fields that appear for all incidents in [Incident Settings][2].
 
 ### Creating an incident
 
-You can declare an incident directly from a graph. You can also go to the upper right hand side of the Incidents page and click on “New Incident”. The following dialog box appears.
+#### From a graph
+
+You can declare an incident directly from a graph by clicking the export button on the graph and then clicking **Declare incident**. The incident creation modal appears, and the graph is added to the incident as a signal. 
+
+{{< img src="monitors/incidents/from-a-graph.png" alt="Create in incident from a graph">}}
+
+#### From the clipboard
+
+Use the Datadog Clipboard to gather multiple monitors and graphs and to generate an incident. To add a dashboard to the clipboard, copy any graph, and then select **Open Clipboard**. Add all of the relevant graphs and monitors to the clipboard and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.
```

Comment:
Use the Datadog Clipboard to gather multiple signals like monitors, graphs, security signals, logs, or any data in Datadog and to generate an incident. To add a graph to the clipboard, copy any graph, and then select **Open Clipboard**. Continuing adding all of the relevant graphs and monitors to the clipboard, select them, and then click **Add to New Incident**. Everything on the clipboard is added to the incident as a signal.

You can also add these signals to active incidents by scrolling down and selecting the incident ID. 

Note: the Datadog Clipboard is in beta

We should also link to docs